osm2streets-js/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "osm2streets-npm"
+version = "0.1.0"
+description = "Javascript API for osm2streets"
+repository = "https://github.com/a-b-street/osm2streets"
+license = "Apache-2.0"
+edition = "2021"
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+
+[features]
+default = ["console_error_panic_hook"]
+
+[dependencies]
+abstutil = { git = "https://github.com/a-b-street/abstreet" }
+console_error_panic_hook = { version = "0.1.6", optional = true }
+import_streets = { path = "../import_streets" }
+# TODO Upstream this in abstutil crate. WASM is missing some runtime dep otherwise.
```

Comment:
Without this, I was hitting https://github.com/rustwasm/wasm-bindgen/issues/2215. I followed the `wasm2wat` workflow in there and figured out this was the problem.